nl/guides/course-management/README.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Cursusbeheer
+description: "Tutorial: cursusbeheer"
+---
+
+# Cursusbeheer
+
+**Inhoudsopgave**
+[[toc]]
+
+Een cursus wordt opgebouwd als een [leerpad](#leerpad) met oefeningen die gebundeld worden in oefeningenreeksen. De opeenvolging van oefeningenreeksen impliceert een mogelijke volgorde waarin de oefeningen kunnen opgelost worden.
+
+## Beheersrechten
+
+Binnenin een cursus kunnen de volgende beheersrechten toegekend worden:
+
+ | naam                                          | icoontje                                |permissies|
+ | ----------------------------------------------|-----------------------------------------|----------|
+ | `Student`|   *geen*|                                  studenten kunnen hun gebruikersprofiel bewerken, kunnen zich registreren voor cursussen en kunnen oplossingen indienen voor oefeningen|
+|`Cursusbeheerder`| ![cursusbeheerder](../../../images/role_icons/staff.png)|cursusbeheerders krijgen alle permissies van studenten en kunnen bovendien ook cursusbeheerders aanduiden, oplossingen van studenten bekijken en de cursus zelf bewerken| 
+
+## Cursus aanmaken
+
+Voor informatie over hoe je een nieuwe cursus aanmaakt, zie deze [pagina](../new-course).
```

Comment:
Move this sentence to the end of the intro paragraph of the page and remove the title.